src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,34 @@
+package tasks;
+
+public abstract class Task {
+    private final String name;
+    private boolean done;
+
+    public Task (String name) {
+        this.name = name;
+        this.done = false;
+    }
+
+    @Override
+    public String toString() {
+        if (done) {
+            return "[x] " + this.name;
+        }
+        return "[ ] " + this.name;
+    }
+
+    public boolean isDone() {
+        return done;
+    }
+
+    public void setDone() {
+        this.done = true;
+    }
+
+    public void setUndone() {
+        this.done = false;
+    }
+
+    public String getName() { return this.name;
```

Comment:
Perhaps moving the `return` statement into another line below?

Reply:
Agreed! Thank you!